portfolio/src/main/webapp/index.html

```diff
@@ -82,7 +82,7 @@ <h1>About Me</h1>
       creating art, baking pastries, and trying new things. </p>
       </div>
         <img src="./image/red.jpg" alt="">
-        <div class="secret-message"><div id="message-container"></div></div>
+        <!-- <div class="secret-message"><div id="message-container"></div></div> -->
```

Comment:
Same comment here about deleting unused things rather than just commenting it out because you can always find it in your commit history later to re-add if you want to.